server/app/src/androidTest/java/sh/calaba/instrumentationbackend/StatusReporter.java

```diff
@@ -32,21 +26,8 @@ public void reportFailure(Throwable e) {
         reportFailure(stringWriter.toString());
     }
 
-    public void clear() {
-        Intent intent = new Intent(context, StatusReporterActivity.class);
-        intent.putExtra(StatusReporterActivity.EXTRA_METHOD, "clear");
-        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-        context.startActivity(intent);
-    }
-
-    public enum FinishedState {SUCCESSFUL, NOT_SUCCESSFUL};
-
-    public void reportFinished(FinishedState state)  {
-        Intent intent = new Intent(context, StatusReporterActivity.class);
-        intent.putExtra(StatusReporterActivity.EXTRA_METHOD, "report-finished");
-        intent.putExtra(StatusReporterActivity.EXTRA_STATE, state);
-        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-        context.startActivity(intent);
```

Comment:
Why did we remove these two methods?
I think it may cause security issues, because android devices won't clean up properly

Reply:
Oh, i see deleting these files via shell commands in [this PR](https://github.com/xamarin/Xamarin.UITest/pull/1487). But it looks like it will work only on Xamarin.UITest. What about tests running on Calabash framework?

Reply:
I should check, but I believe that these files are only consumed by Xamarin framework. Calabash itself may rely on finish(RESULT_OK) method, but I will check